.github/workflows/release-build.yml

```diff
@@ -52,6 +53,67 @@ jobs:
           GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
 
+      - name: Create checksum
+        run: |
+          echo ORIG_CHECKSUM="$(shasum otelagent/build/libs/aws-opentelemetry-agent-*.jar | awk '{ print $1 }')" >> "$GITHUB_ENV"
+
+      - name: Get current version
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        shell: bash
+        run: |
+          echo ADOT_JAVA_VERSION="$(./gradlew printVersion -q )" >> $GITHUB_ENV
```

Comment:
same here. the version should come from `${{ github.event.inputs.version }}`. Please check https://github.com/aws-observability/aws-otel-java-instrumentation/pull/358/files#diff-52120b4145bd2cf9c735193f6a093a7944688b21f04e854447e34a5049fc829fR48